src/sentry/tempest/tasks.py

```diff
@@ -146,9 +146,8 @@ def poll_tempest_crashes(credentials_id: int, **kwargs) -> None:
                     project_id=project_id, trigger="tempest:poll_tempest_crashes"
                 )
 
-            # Check if we should attach screenshots  and or dumps (opt-in features)
+            # Check if we should attach screenshots (opt-in feature)
             attach_screenshot = credentials.project.get_option("sentry:tempest_fetch_screenshots")
-            attach_dump = credentials.project.get_option("sentry:tempest_fetch_dumps")
```

Comment:
in this case, do we still need to have the project option "sentry:tempest_fetch_dumps" around? if not, we need to do a cleanup